### PR TITLE
Get-Post requests on projecthunt api

### PR DESCRIPTION
**Pull request by rutikwankhade**

### CODE REVIEW

File: src/components/AddNewProject.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
 import React from "react";
 import { useForm } from "react-hook-form";
+import axios from 'axios';
 
 const AddNewProject = () => {
     const { register, errors, handleSubmit } = useForm();
 
-    const onSubmit = data => console.log(data);
+    const onSubmit = (data) => {
+        // console.log(data);
```
Comment (sagarchoudhary96) [inline src/components/AddNewProject.js:9]: remove commented code, if not needed.

File: src/components/AddNewProject.js [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+            "projectName": data.projectName,
+            "description": data.description,
+            "category": ["Frontend"],
+            "language": tech,
+            "userId": "1"
+        }
+
+        console.log(project);
+
+        axios.post('https://projecthunt-api.herokuapp.com/project', project)
```
Comment (sagarchoudhary96) [inline src/components/AddNewProject.js:22]: create a new file under `utils` directory, for e.g `api.js` and all the API calls can be moved there and you can export the function such as `createProject` which can take in data parameter and do API call over there and return a promise.

File: src/pages/ProjectShowcase.js [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
 import ProjectCard from '../components/ProjectCard';
 import Dropdown from '../components/Dropdown';
+import loaderIcon from 'assets/images/loader.svg';
+
 
 const ProjectShowcase = () => {
     const options = ['Top Voted', 'Recently added', 'Featured']
 
     const [selected, setSelected] = useState(options[0]);
+    const [fetchedProjects, setFetchedProjects] = useState([]);
```
Comment (sagarchoudhary96) [inline src/pages/ProjectShowcase.js:11]: can be given more meaningful name such as `allProjects` or `projectsList`

File: src/pages/ProjectShowcase.js [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
+        setLoading(true);
+        fetch('https://projecthunt-api.herokuapp.com/project')
+            .then(res => res.json())
+            .then(data => {
+                setFetchedProjects(data.projects)
+                setLoading(false);
+                console.log(data);
+            })
+            .catch(error => { console.log(error) })
+    }, [])
```
Comment (sagarchoudhary96) [inline src/pages/ProjectShowcase.js:25]: this too can be moved to api utils.

**Comment by dipakkr** [comment]
@rutikwankhade any update on changes ?

File: src/utils/api.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+    return response;
+}
+
+function createProject(project) {
+    axios.post('https://projecthunt-api.herokuapp.com/project', project)
+        .then(response => console.log(response))
+        .catch(error => console.log(error))
+}
+
```
Comment (dipakkr) [inline src/utils/api.js:15]: pls, maintain es6 standards. You can use the arrow function instead of this function.

**Review by dipakkr — APPROVED** [review]

**Pull request merged**